iDiff/differs/package.go

```diff
@@ -4,20 +4,39 @@ import (
 	"bytes"
 	"fmt"
 	"os"
-	"runtimes-common/iDiff/utils"
+
+	"github.com/golang/glog"
+	"github.com/runtimes-common/iDiff/utils"
 )
 
-//  Diffs two packages to see if they have the same contents
-func Package(d1file, d2file string) string {
+//  Diffs two packages and compares their contents
+func Package(img1, img2 string) (string, error) {
+	dir1, path1, err := utils.ImageToDir(img1)
+	if err != nil {
+		return "", err
+	}
+	dir2, path2, err := utils.ImageToDir(img2)
+	if err != nil {
+		return "", err
+	}
+	diff := getDiffOutput(dir1, dir2)
+
+	defer os.RemoveAll(path1)
+	defer os.RemoveAll(path2)
+	defer os.Remove(dir1)
+	defer os.Remove(dir2)
+
+	return diff, nil
+}
+
+func getDiffOutput(d1file, d2file string) string {
 	d1, err := utils.GetDirectory(d1file)
 	if err != nil {
-		fmt.Printf("Error reading directory structure from file %s: %s\n", d1file, err)
-		os.Exit(1)
+		glog.Errorf("Error reading directory structure from file %s: %s\n", d1file, err)
```

Comment:
Errorln() doesn't allow formatting, so I'd have to break up the text a bit to pass formatted parts as different arguments to Errorln, but is that preferred over this?